src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -267,6 +269,33 @@ public void moveToDefaultDirectory() {
         }
     }
 
+    /**
+     * Gets the filename for the current linked file and compares it to the new suggested filename.
+     * @return true if the suggested filename is same as current filename.
+     */
+    public boolean isGeneratedNameSameAsOriginal() {
+
```

Comment:
Please remove these empty lines at the start of the method.

Reply:
Done. 👍